src/containers/Dashboard/Dashboard.tsx

```diff
@@ -1,50 +1,47 @@
 import React from 'react';
-import Box from '@mui/material/Box';
 import './styles.scss';
 import ProjectTile from '../../components/ProjectTile/ProjectTile';
-
-const cards = [
-  {
-    id: 1,
-    name: 'Aplikacja mobilna mbank',
-    activeTasks: 26,
-    progressBar: 76,
-    typeOfProject: 'Aplikacja',
-    endDate: '26.01.2022',
-  },
-  {
-    id: 2,
-    name: 'Aplikacja mobilna mbank',
-    activeTasks: 26,
-    progressBar: 76,
-    typeOfProject: 'Aplikacja',
-    endDate: '26.01.2022',
-  },
-  {
-    id: 3,
-    name: 'Aplikacja mobilna mbank',
-    activeTasks: 26,
-    progressBar: 76,
-    typeOfProject: 'Aplikacja',
-    endDate: '26.01.2022',
-  },
-  {
-    id: 4,
-    name: 'Aplikacja mobilna mbank',
-    activeTasks: 26,
-    progressBar: 76,
-    typeOfProject: 'Aplikacja',
-    endDate: '26.01.2022',
-  },
-];
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getProjects, selectLoadingProjects, selectProjects } from 'redux/project/project.slice';
+import { selectAccessToken } from '../../redux/auth/auth.slice';
+import { myProjectsType } from 'core/types/requests/project.types';
 
 const MainLayout = () => {
+  const apiToken = useSelector(selectAccessToken);
+  const entities: myProjectsType = useSelector(selectProjects);
+  const loading = useSelector(selectLoadingProjects);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getProjects(apiToken));
+  }, [apiToken, dispatch]);
+
+  if (loading) return <p>Loading...</p>;
+
+  const cards = entities.projectsList?.map((project: any) => {
+    return {
+      id: project.id,
+      name: project.name,
+      activeTasks: 26,
+      progressBar: project.progressPercentage,
+      typeOfProject: 'Aplikacja',
+      endDate: project.dueDate,
+    };
+  });
```

Comment:
Tutaj jak rozumiem wyciągnięcie projektów bo tam jeszcze count był podany, ogólnie to możesz już w redusie pominąć tego counta bo i tak paginacji nie mamy a jak by była to wtedy się pomyśli